supported-standards/lens-open-algorithm-standards/models/publication/feed/post.feed.model.md

```diff
@@ -0,0 +1,106 @@
+# Model
+
+## Name
+
+Posts Feed
+
+### Personal
+
+Supported - `post_personal_feed`
+
+### Global
+
+Supported - `post_global_feed`
+
+## Data Output
+
+### Personal
+JSON schema can be found [here](./personal-schema.json) for usage in json validation.
+
+```json
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://lens.xyz/post.feed.schema.json",
+  "title": "Post Feed",
+  "description": "Ordered list of posts to be shown to users",
+  "type": "object",
+  "properties": {
+    "profileId": {
+      "type": "string",
+      "description": "ProfileId for which the posts are being recommended",
+      "pattern": "^0x[0-9a-fA-F]+$"
+    },
+    "posts": {
+      "description": "array of postIds",
+      "type": "array",
+      "items": {
+        "type": "string",
+        "description": "postId",
+        "pattern": "^0x[0-9a-fA-F]+$"
+      },
+      "minItems": 100,
+      "maxItems": 1000
+    }
+  },
+  "required": ["profileId", "posts"]
+}
+```
+
+Example:
+
+```json
+{
+  "profileId": "0x01", 
+  "posts": ["0x01", "0x02"]
+}
+```
+
+
+#### Rules
+
+Enter rules here
+
+### Global
+JSON schema can be found [here](./global-schema.json) for usage in json validation.
+
+```json
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://lens.xyz/post.feed.schema.json",
+  "title": "Post Feed",
+  "description": "Ordered list of posts to be shown to users",
+  "type": "object",
+  "properties": {
+    "posts": {
+      "description": "array of postIds",
+      "type": "array",
+      "items": {
+        "type": "string",
+        "description": "postId",
+        "pattern": "^0x[0-9a-fA-F]+$"
+      },
+      "minItems": 100,
+      "maxItems": 1000
+    }
+  },
+  "required": ["posts"]
+}
+```
+
+Example:
+
+```json
+{
+  "posts": ["0x01", "0x02"]
```

Comment:
this should use the internal publication id the API does really which is {posterProfileId}-{publicationId} i don't think your regex works on cases like 0x01-0x01 else you wont know how to load it.

Reply:
Will fix the regex and examples.